store/src/java/com/zimbra/cs/dav/resource/ScheduleOutbox.java

```diff
@@ -173,6 +178,10 @@ public void handlePost(DavContext ctxt) throws DavException, IOException, Servic
             }
         }
 
+        if (Collections.list(recipients).size() == 0) {
```

Comment:
```suggestion
        if (!recipients.hasMoreElements()) {
```
This should do the same thing, but readability is a toss-up. Not sure why they specifically use an Enumeration here.

Reply:
Done